psi4/src/psi4/libcubeprop/cubeprop.cc

```diff
@@ -154,6 +158,52 @@ void CubeProperties::raw_compute_properties() {
             }
             if (indsa0.size()) compute_orbitals(Ca_, indsa0, labelsa, "Psi_a");
             if (indsb0.size()) compute_orbitals(Cb_, indsb0, labelsb, "Psi_b");
+        } else if (task == "FRONTIER_ORBITALS") {
+            std::vector<int> indsa0;
+            std::vector<int> indsb0;
+            std::vector<std::string> labelsa;
+            std::vector<std::string> labelsb;
+            CharacterTable ct = basisset_->molecule()->point_group()->char_table();
+            if (nalpha_ == nbeta_) {
+                indsa0.push_back(nalpha_-1);
+                labelsa.push_back(to_string(std::get<1>(info_a_[nalpha_-1]) + 1) + "-" +
```

Comment:
We can use `std::to_string` these days. Should probably try to kill off the old Psi `to_string` at some point.